tests/monitoring_integration_test.py

```diff
@@ -174,6 +208,46 @@ def testFrontendServiceSloExists(self):
 		found_latency_slo = self.checkForSlo('frontend-srv', 'latency-slo')
 		self.assertTrue(found_latency_slo)
 
+	def testCartServiceSloExists(self):
+		""" Test that for each service that two SLOs (availability, latency) get created. """
+		cartservice_id = 'ist:' + project_id + '-zone-' + zone + '-stackdriver-sandbox-default-cartservice'
```

Comment:
can you encapsulate this in a method and call it with servicename as parameter across all the test methods?

Reply:
yup! changed :)